engine/app/controllers/good_job/assets_controller.rb

```diff
@@ -35,11 +35,11 @@ def rails_ujs_js
     end
 
     def scripts_js
-      render file: GoodJob::Engine.root.join("app", "assets", "scripts.js")
+      render file: GoodJob::Engine.root.join("app", "assets", "good_job", "scripts.js")
     end
 
     def style_css
-      render file: GoodJob::Engine.root.join("app", "assets", "style.css")
+      render file: GoodJob::Engine.root.join("app", "assets", "good_job", "style.css")
```

Comment:
I went ahead and namespaced these as well, though they weren't conflicting with any of _our_ assets as far as I can tell.